masonry/src/doc/06_masonry_concepts.md

```diff
@@ -100,6 +100,26 @@ A widget is considered "interactive" if it can still get text and/or pointer eve
 Stashed and disabled widget are non-interactive.
 
 
+## Layout rect
+
+A widget's layout rect includes its self-declared size, and the position attributed by its parent (local position) / parents (global position).
+
+Pointer events cannot target a widget if they are outside its layout rect, though they can target its children (see **Bounding rect** section).
+
+A layout rect doesn't necessarily have a formal definition; it's generally where the widget will be drawn, though a widget can be drawn outside its layout rect; it's usually a widget's own space and containers will try to avoid having its children's layout rects overlap.
```

Comment:
I think this section should somehow explain, that a widget can be transformed by its transform.

Pointer events also only reach the widget, when they intersect the *transformed* widget, thus I don't think it's currently correct. Try for example transformations on the left portal in the xilem http_cats example, to see what I mean.